azure/src/main/scala/quasar/blobstore/azure/reactive.scala

```diff
@@ -20,18 +20,14 @@ import cats.effect._
 import fs2.Stream
 import fs2.interop.reactivestreams._
 import reactor.core.publisher.{Flux, Mono}
-import reactor.core.scala.publisher.ScalaConverters._
 
 object reactive {
 
   def streamToFlux[F[_]: ConcurrentEffect, A](s: Stream[F, A]): Flux[A] =
     Flux.from(s.toUnicastPublisher)
 
-  def monoToAsync[F[_]: ContextShift, A](
-      mono: Mono[A])(
-      implicit F: Async[F])
-      : F[A] =
-    Async.fromFuture(F.delay(mono.asScala.toFuture))
+  def monoToAsync[F[_]: ConcurrentEffect, A](mono: Mono[A]): F[A] =
+    fromPublisher[F, A](mono).compile.lastOrError
```

Comment:
This is probably the most significant change. We used to use [this](https://github.com/spring-attic/reactor-scala-extensions) to convert, but it looks obsolete given that Scala has supported Java lambdas since 2016.

This appeared to be the most straightforward way to convert `Mono`s to `IO`, given `Mono`s are also publishers.